mule-apikit-module/src/test/resources/org/mule/module/apikit/com/mycompany/test-api/1.0.0/test-api-1.0.0.pom

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is necessary?

Reply:
Yes it is this is for integration testing (no automated yet)